renderer/reducers/invoice.js

```diff
@@ -244,6 +246,9 @@ const ACTION_HANDLERS = {
     state.isInvoicesLoading = false
     state.createInvoiceError = null
   },
+  [SET_OFFSET]: (state, { offset }) => {
+    state.offset = offset
```

Comment:
new state prop added here but not defined in `initialState`

Reply:
This is a leftover and was removed in the update